client/verta/verta/_registry/model.py

```diff
@@ -37,12 +41,46 @@ def _generate_default_name(cls):
 
     @classmethod
     def _get_proto_by_id(cls, conn, id):
-        raise NotImplementedError
+        Message = _RegisteredModelService.GetRegisteredModelRequest
+        response = conn.make_proto_request("GET",
+                                           "/api/v1/registry/registered_models/{}".format(id))
+        return conn.maybe_proto_response(response, Message.Response).registered_model
 
     @classmethod
     def _get_proto_by_name(cls, conn, name, workspace):
-        raise NotImplementedError
+        Message = _RegisteredModelService.GetRegisteredModelRequest
+        if name and workspace :
+            response = conn.make_proto_request("GET",
+                                               "/api/v1/registry/workspaces/{}/registered_models/{}".format(workspace, name))
+        else:
+            raise RuntimeError("the Client has encountered an error;"
+                               " please notify the Verta development team: registered model name or workspace not specified")
```

Comment:
I think we do not need this `if-else` check, because `name` and `workspace` should be valid when they are passed into this function.
```suggestion
        response = conn.make_proto_request("GET",
                                           "/api/v1/registry/workspaces/{}/registered_models/{}".format(workspace, name))
```